src/routers/router.ts

```diff
@@ -182,3 +183,10 @@ export abstract class ISwapToRatio<RoutingConfig, SwapAndAddConfig> {
     routingConfig?: RoutingConfig
   ): Promise<SwapToRatioResponse>;
 }
+
+export class StaticGasPriceProvider implements IGasPriceProvider {
+  constructor(private gasPriceWei: BigNumber) {}
+  async getGasPrice(): Promise<GasPrice> {
+    return { gasPriceWei: this.gasPriceWei };
+  }
+}
```

Comment:
er - yes, we still need this because we have a specific test where we specify the gas price in wei and create the alpha router with that specific gasPriceProvider: https://github.com/Uniswap/smart-order-router/pull/95/files#diff-b1a09e5a7156f19225eba6ca4e2e78d0942580ff1118bab472bf0ea3b1294904R926-R933